File: internal/content/testdata/bench/go/top-two-elements-sum.go
func top_two_sum(nums []int) int {
	first, second := 0, 0
	for _, x := range nums {
		if x >= first {
			second = first
			first = x
		} else if x > second {
			second = x
		}
	}
	return first + second
}

File: internal/content/testdata/bench/javascript/top-two-elements-sum.js
function top_two_sum(nums) {
  let first = 0, second = 0;
  for (const x of nums) {
    if (x >= first) {
      second = first;
      first = x;
    } else if (x > second) {
      second = x;
    }
  }
  return first + second;
}
